frontend-ssr/pages/api/user/login.js

```diff
@@ -0,0 +1,38 @@
+//Imports
+const User = require("../../../models/User.schema");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const { loginValidation } = require("../../../utils/validation/userValidation")
+
+//Vars
+const error_message = { status: "fail", message: 'Email or Password is incorrect' }
```

Comment:
Let's name this something a little less generic. `error_message` doesn't tell you what kind of error or situation are we in.

Reply:
Another mention I just remembed. Let's CAPS_LOCK constant names so we're all on the same page with the styling.